hack/run_test

```diff
@@ -506,6 +531,7 @@ function run_general_tests() {
   DOCKER_ARGS="-u 12345" PGUSER=user2 PASS=pass run_tests no_admin_altuid
   DOCKER_ARGS="-u 12345" PGUSER=user3 PASS=pass1 ADMIN_PASS=r00t run_tests admin_altuid
   DB=postgres DOCKER_ARGS="-u 12345" ADMIN_PASS=rOOt run_tests only_admin_altuid
+  run_doc_check
```

Comment:
In case this function stays in postgresql-container sources, can we have it in `TEST_LIST=`